src/components/FloatingHelper/content/HelperContent.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import style from './HelperContent.st.css';
+import {Text} from '../../Text';
+import {DataHooks} from './DataHooks';
+
+export interface HelperContentProps {
+  /** Adds text as the title */
+  title?: string;
+  text?: string;
+}
+
+export const HelperContent: React.SFC<HelperContentProps> = (props: HelperContentProps) => {
+  return (
+    <div {...style('root', {}, props)}>
+      {props.title &&
+        <div className={style.title}>
+          <Text dataHook={DataHooks.title} appearance="T2.2" light>
```

Comment:
You are using `appearance` but this is a deprecated API: https://github.com/wix/wix-ui-backoffice/blob/7e16882e4855c073c4afe93b90c075303f2719ed/src/components/Text/Text.tsx#L8

Reply:
Follow the migration guide if you want in wix-style-react for next version 4 branch

Reply:
Thanks. I was wondering why the appearance is wrong in the storybook.
I found out that `createHOC` returns type `any` so we are loosing all types, and we don't get props compile-time validation.
I fixed the Text props here, and added a task to fix the `createHOC`.